### PR TITLE
{vis}[foss/2019a,intel/2019a] VTK v8.2.0 w/ Python 3.7.2

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by furstj** [comment]
Test report by @furstj
**SUCCESS**
Build succeeded for 5 out of 5 (2 easyconfigs in this PR)
pc-12101-fuji - Linux UNKNOWN UNKNOWN, Intel(R) Xeon(R) CPU E3-1241 v3 @ 3.50GHz, Python 2.7.14
See https://gist.github.com/fe4a2721f58ba8fcf4fa2e7aaa8667b8 for a full test report.

File: easybuild/easyconfigs/v/VTK/VTK-8.2.0-intel-2019a-Python-3.7.2.eb [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+# Python
+configopts += "-DVTK_WRAP_PYTHON=ON "
+configopts += "-DVTK_PYTHON_VERSION=%(pyver)s "
+configopts += "-DPYTHON_INCLUDE_DIR=$EBROOTPYTHON/include/python%(pyshortver)sm "
+configopts += "-DPYTHON_LIBRARY=$EBROOTPYTHON/lib/libpython%%(pyshortver)sm.%s " % SHLIB_EXT
+# Other
+configopts += "-DVTK_Group_MPI:BOOL=ON "
+configopts += "-DCMAKE_INSTALL_LIBDIR=lib"
+
+# preinstallopts = "mkdir -p %(installdir)s/lib/python%(pyshortver)s/site-packages/ && "
```
Comment (boegel) [inline easybuild/easyconfigs/v/VTK/VTK-8.2.0-intel-2019a-Python-3.7.2.eb]: @Micket Let's remove this commented out line?

Comment (Micket) [reply easybuild/easyconfigs/v/VTK/VTK-8.2.0-intel-2019a-Python-3.7.2.eb]: Fixed

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2567.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/96b7422b140bc422a388e077a87aa62d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3147.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/376c503d1c1bacf0b0d1e302f700cb4e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2026.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/7cf297281c8e739c39369438a38acb79 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @Micket!

**Pull request merged**